server/subscription/subscription.go

```diff
@@ -0,0 +1,64 @@
+package subscription
+
+import "strings"
+
+type Subscription struct {
```

Comment:
3/5 that this shouldn't be exported, if you have a `New()` method.

Reply:
I can't because New() return a `*Subscription`, if I made it private, all code can't build

![Sélection_222](https://user-images.githubusercontent.com/1492516/56689308-148a7500-66db-11e9-8b5d-d4452581689e.png)

Maybe i have not understand something!?

Reply:
NVM, I didn't noticed that `Subscription` is used outside of the package.

Reply:
:ok_hand: